modules/mobile/spec/request/prescriptions_request_spec.rb

```diff
@@ -49,6 +94,21 @@
       end
     end
 
+    context 'when cache is populated' do
+      it 'uses cache instead of service' do
+        path = Rails.root.join('modules', 'mobile', 'spec', 'support', 'fixtures', 'prescriptions.json')
```

Comment:
Given that we're doing this twice now, you could also move this into a method called `set_cache`. Nothing wrong with making helper methods in your specs IMO.